lib/screens/component/tm_elevated_button.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:go_router/go_router.dart';
+import 'package:room_master_app/common/extensions/context.dart';
+import 'package:room_master_app/l10n/l10n.dart';
+
+import '../../blocs/authentication/authentication_cubit.dart';
+import '../../navigation/navigation.dart';
+
+class TMElevatedButton extends StatelessWidget {
+  const TMElevatedButton({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: double.infinity,
+      height: 50,
```

Comment:
Đừng để height mặc định
cho nó thành tham số kẻo lỡ có nút to có nút nhỏ
`final double? height`
`...`
`height: height ?? 50`
kiểu kiểu rứa